authors/apps/articles/tests/test_create_article.py

```diff
@@ -41,3 +43,25 @@ def test_api_cannot_create_an_article_with_an_unauthenticated_user(self):
         )
 
         self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+    def test_that_every_256_words_or_below_is_1_min_read(self):
```

Comment:
I see here that you have 256 yet in the implementation it is 265. Kindly elaborate on which it is and fix accordingly

Reply:
Its 256 I had implemented it with a mistake